folksonomy/models.py

```diff
@@ -24,8 +24,10 @@ class ProductTag(BaseModel):
 
     @validator('product')
     def product_check(cls, v):
+        if v == '':
+            raise ValueError('barcode cannot be empty')
```

Comment:
ok, if we change regexp as propose above we do not necessarily need it, but it's good to have a more precise message.

Reply:
I thought a precise message would be more apt. So, I'll keep the regexp as it is.